src/main/java/org/dependencytrack/model/Project.java

```diff
@@ -138,6 +138,7 @@
         })
 })
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@Index(name = "PROJECT_ACTIVE_IDX", members = {"project", "active"})
```

Comment:
`project` is not a valid field here.

```suggestion
@Index(name = "PROJECT_ACTIVE_IDX", members = {"id", "active"})
```

Reply:
Good catch, fixed.